ja2003.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```

Comment:
Has this been submitted upstream?
Also ideally it would have the proper name from start

Reply:
No. This is the same as for the AppData. "I am upstream" and this desktop file is specific for the flatpak.

Reply:
desktop file are NOT specific to flatpak either.

Reply:
Sorry, I mean that I created this very desktop file just for flatpak, because of the icon and executable path.

Reply:
FTR, these can be re-used by any Linux distro. Files can be easily modified with `desktop-file-edit` to set proper values. This is not a blocker, though.